--- kmscon-kmscon-7/Makefile.am~	2013-03-09 21:41:00.000000000 +0200
+++ kmscon-kmscon-7/Makefile.am	2013-03-09 21:42:16.711615577 +0200
@@ -56,16 +56,6 @@
 AM_LDFLAGS = \
 	-Wl,--as-needed
 
-if BUILD_ENABLE_DEBUG
-AM_CFLAGS += -g
-endif
-
-if BUILD_ENABLE_OPTIMIZATIONS
-AM_CFLAGS += -O2
-else
-AM_CFLAGS += -O0
-endif
-
 #
 # SHL - Static Helper Library
 # The SHL subsystem contains several small code pieces used all over kmscon and
