--- kmscon-5/src/uterm_input_uxkb.c.orig	2012-09-11 19:14:56.837808737 +0200
+++ kmscon-5/src/uterm_input_uxkb.c	2012-11-09 16:37:55.362089827 +0100
@@ -298,7 +298,9 @@
 
 	/* TODO: fix xkbcommon upstream to be case-insensitive if case-sensitive
 	 * match fails. */
-	keysym = xkb_keysym_from_name(n);
+	keysym = xkb_keysym_from_name(n, 0);
+	if (!keysym)
+		keysym = xkb_keysym_from_name(n, XKB_KEYSYM_CASE_INSENSITIVE);
 	if (!keysym)
 		return -EFAULT;
 
