--- kmscon-5/src/genunifont.c.orig	2012-08-25 15:54:35.997327647 +0200
+++ kmscon-5/src/genunifont.c	2012-11-09 20:21:20.869890307 +0100
@@ -82,10 +82,10 @@
 
 	idx = hex_val(c);
 	if (idx < 16) {
-		fprintf(out, line_map[idx]);
+		fprintf(out, "%s", line_map[idx]);
 	} else {
 		fprintf(stderr, "genunifont: invalid value %c\n", c);
-		fprintf(out, line_map[0]);
+		fprintf(out, "%s", line_map[0]);
 	}
 }
 
