--- kmscon-kmscon-7/Makefile.am~	2013-03-09 21:41:00.000000000 +0200
+++ kmscon-kmscon-7/Makefile.am	2013-03-09 21:42:16.711615577 +0200
@@ -56,16 +56,6 @@
 AM_LDFLAGS = \
 	-Wl,--as-needed
 
-if BUILD_ENABLE_DEBUG
-AM_CFLAGS += -g
-endif
-
-if BUILD_ENABLE_OPTIMIZATIONS
-AM_CFLAGS += -O2
-else
-AM_CFLAGS += -O0
-endif
-
 #
 # SHL - Static Helper Library
 # The SHL subsystem contains several small code pieces used all over kmscon and
@@ -198,7 +198,7 @@
 	external/dbus-loop.h \
 	external/dbus-loop.c
 libeloop_la_CPPFLAGS += $(DBUS_CFLAGS)
-libeloop_la_LIBADD += $(DBUS_LIBS)
+libeloop_la_LIBADD += $(DBUS_LIBS) -lpthread
 endif
 
 #
@@ -233,6 +233,7 @@
 	$(XKBCOMMON_CFLAGS)
 libtsm_la_LIBADD = \
 	$(XKBCOMMON_LIBS) \
+	-lrt \
 	libshl.la
 EXTRA_libtsm_la_DEPENDENCIES = ${top_srcdir}/docs/sym/libtsm.sym
 libtsm_la_LDFLAGS = \
