--- kmscon-6/Makefile.am~	2013-03-09 20:27:31.000000000 +0200
+++ kmscon-6/Makefile.am	2013-03-09 20:27:42.798733628 +0200
@@ -56,16 +56,6 @@
 AM_LDFLAGS = \
 	-Wl,--as-needed
 
-if BUILD_ENABLE_DEBUG
-AM_CFLAGS += -g
-endif
-
-if BUILD_ENABLE_OPTIMIZATIONS
-AM_CFLAGS += -O2
-else
-AM_CFLAGS += -O0
-endif
-
 #
 # SHL - Static Helper Library
 # The SHL subsystem contains several small code pieces used all over kmscon and
@@ -112,7 +112,7 @@
 libeloop_la_CPPFLAGS += \
 	$(DBUS_CFLAGS)
 libeloop_la_LIBADD += \
-	$(DBUS_LIBS)
+	$(DBUS_LIBS) -lpthread
 endif
 
 #
@@ -156,6 +146,7 @@
 	$(AM_CPPFLAGS)
 libtsm_la_LDFLAGS = \
 	-version-info 1:0:0
+libtsm_la_LIBADD = -lrt
 
 #
 # libuterm
