--- kmscon-5/Makefile.am.orig	2012-09-26 22:56:09.716902778 +0200
+++ kmscon-5/Makefile.am	2012-11-09 17:56:46.818557932 +0100
@@ -56,16 +56,6 @@
 AM_LDFLAGS = \
 	-Wl,--as-needed
 
-if BUILD_ENABLE_DEBUG
-AM_CFLAGS += -g
-endif
-
-if BUILD_ENABLE_OPTIMIZATIONS
-AM_CFLAGS += -O2
-else
-AM_CFLAGS += -O0
-endif
-
 #
 # SHL - Static Helper Library
 # The SHL subsystem contains several small code pieces used all over kmscon and
@@ -119,7 +109,7 @@
 	$(AM_CPPFLAGS) \
 	$(DBUS_CFLAGS)
 libeloop_la_LIBADD = \
-	$(DBUS_LIBS)
+	$(DBUS_LIBS) -lpthread
 libeloop_la_LDFLAGS = \
 	-version-info 1:0:0
 
@@ -156,6 +146,7 @@
 	$(AM_CPPFLAGS)
 libtsm_la_LDFLAGS = \
 	-version-info 1:0:0
+libtsm_la_LIBADD = -lrt
 
 #
 # libuterm
